src/script/components/MessagesList/Message/SystemMessage/SystemMessageBase.tsx

```diff
@@ -39,7 +39,9 @@ export const SystemMessageBase: React.FC<SystemMessageProps> = ({message, isSend
       <p className="message-header-label">
         <span className="message-header-label__multiline">
           {isSenderNameVisible && <span className="message-header-sender-name">{unsafeSenderName}</span>}
-          {message.caption && <span className="ellipsis">{message.caption}</span>}
+          {message.caption && (
+            <span className="system-message-caption ellipsis" dangerouslySetInnerHTML={{__html: message.caption}} />
+          )}
```

Comment:
**out of scope**: This is probably not for this PR, but in the future, we should try to move this caption for the `entity` to the `react component` layer and render it a pure React node (not `innerHTML`)